### ISSUE TITLE
Maintenance status ?

### ISSUE THREAD

**Issue by rzr** [kelly/node-i2c#94]
Is this project still maintained here or in forks ?

If it is abandoned, may I suggest to transfer it to this org:

https://github.com/abandonware

See procedure:

https://abandonware.github.io/

**Comment by kelly** [comment]
I'm not going to transfer. Submit a PR if you have an update to the project.

**Comment by rzr** [comment]
good to have feedback ! may you review those pending changes then:

https://github.com/kelly/node-i2c/pulls

Regarding creating new PR, I am somehow blocked by this issue:
https://github.com/kelly/node-i2c/issues/91

**Comment by kelly** [comment]
Unfortunately I don't have a solution to that issue at this time.
